faasmcli/faasmcli/util/env.py

```diff
@@ -30,7 +30,7 @@ def _get_dir(variable, default):
 
 FAASM_BUILD_DIR = _get_dir("FAASM_BUILD_DIR", "/build/faasm")
 
-FAASM_CONFIG_FILE = join(FAASM_HOME, "faasm.ini")
+FAASM_CONFIG_FILE = join(PROJ_ROOT, "faasm.ini")
```

Comment:
I think this file is generally outdated, but this is the only required change now.